lib/common/ipc_client.c

```diff
@@ -1332,7 +1332,7 @@ crm_ipc_send(crm_ipc_t *client, const xmlNode *message,
 
     id++;
     CRM_LOG_ASSERT(id != 0); /* Crude wrap-around detection */
-    rc = pcmk__ipc_prepare_iov(id, message, client->max_buf_size, &iov, &bytes);
```

Comment:
FYI I'm pretty sure that with a little rearranging, you could drop the `max_buf_size` member from `struct crm_ipc_s`. I don't think there's any possible weirdness with `qb_ipcc_get_buffer_size()` that would result in overflowing the libqb IPC client's max buffer size or anything. (We call `qb_ipcs_enforce_buffer_size()` with the 128k constant.)

Reply:
Yes, I think you're right.  This also lets me get rid of that block in `pcmk__connect_generic_ipc` that was annoying me in the rest of the split up patches.